### DISCUSSION TITLE
Any ideas of how to copy the vocoding of the Half Life Combine?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [wwmm/easyeffects#2334] [Q&A]
I'm very new to audio manipulation. My ultimate goal is to be able to use the same preset on multiple voices and have those voices be mostly unrecognizable from each other – a kind of vocal anonymity.

**Comment by wwmm** [comment]
The closest thing to this on EasyEffects is the pitch plugin. But it has some limitations and I am not sure if it can mimic whatever this other plugins you used is doing.

**Reply by roke-julian-lockhart** [reply]
> whatever this other plugins you used is doing.

Vocoding, apparently. I don't know what vocoding actually is, but it seems that other software with this capability can mimic it quite well.

https://github.com/wwmm/easyeffects/issues/1401 appears to discuss this, but I don't actually know what they're discussing past what the title states.
